content/webapp/components/IIIFItem/IIIFItem.tsx

```diff
@@ -254,13 +276,15 @@ const IIIFItem: FunctionComponent<ItemProps> = ({
         <Wrapper
           shouldShowItem={shouldShowItem}
           className="item-wrapper"
-          i={i}
+          titleOverride={titleOverride}
           canvas={canvas}
           isRestricted={isRestricted}
         >
           <AudioPlayer
             audioFile={item.id || ''}
```

Comment:
Should we wrap this in a conditional, if there is no `item.id`, don't render it at all?

Reply:
Boolean(item.id) is part of the case conditional, so it wouldn't get this far, the empty string is to keep typescript from complaining